backend/src/controllers/ProjectController.ts

```diff
@@ -264,6 +263,24 @@ class ProjectController {
     ]);
     return userProjects;
   }
+  public async findUserProjectsByTech(
+    userId: string,
+    techId: string
+  ): Promise<ProjectDoc[]> {
+    let userMembers = await this.memberModel.aggregate([
```

Comment:
nit: there's really no reason to do an aggregation here since we only query a single collection here. as much as possible avoid aggregations unless absolutely needed (like when "joining" other collection data).

I would recommend doing this instead:
```ts
const userMembers = this.memberModel.find({ user: userId }).map((m) => m.project);
// The `.map` turns the list of objects into a list of project IDs
...rest
    let userProjects = await this.projectModel.find({
      _id: { $in: userMembers },
      techs: techId,
    });
    ```